src/ButtonWithBadge/ButtonWithBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { Fragment } from 'react';
+import { StyledButton, StyledNewBadge } from './ButtonWithBadge.styles';
+
+export interface ButtonWithBadgeProps {
+  disabled?: boolean;
+  badge?: string;
+  onClick?: () => void;
+}
+
+export const ButtonWithBadge: React.FC<ButtonWithBadgeProps> = ({ children, badge, onClick }) => (
```

Comment:
Not a fan of this being its own thing, should more be a part story. But happy enough to just get it done for now.

Reply:
Ah, that's ok, let's keep it for now and I will make a note to change it later